src/main/java/gg/uhc/uhc/modules/team/requests/RequestManager.java

```diff
@@ -138,6 +138,7 @@ public boolean finalizeRequest(int id, AcceptState accepted) {
 
             for (OfflinePlayer p : players) {
                 team.addPlayer(p);
+                p.setWhitelisted(true);
```

Comment:
A flag on the command would need large refactoriing due to the structure of the request manager. I'll go with the configuration option & adding documention soon.